README.md

```diff
@@ -1,44 +1,71 @@
+* Reduce costs to programmes by sharing infrastructure where possible
+* Minimise vendor lock-in while leveraging managed services by using non-proprietary configuration language and abstractions
+* Improve service team delivery by providing release automation and observability tooling
+* Increase service team confidence in deployments by enabling strong parity between local development and production environments
+* Avoid development bottlenecks and platform stagnation by encouraging all teams to extend and evolve the shared GSP base to meet emerging needs
 
-The GDS Supported Platform is the [strategic container platform](https://reliability-engineering.cloudapps.digital/documentation/strategy-and-principles/re-strategy.html#2-build-a-new-common-hosting-platform-for-gds-services) for hosting services developed for the [Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service).
+GSP provides a suite of pre-configured components along with upstream Kubernetes, much like a GNU/Linux distribution provides a suite of userspace components along with the upstream Linux kernel.
 
-The platform manages the infrastructure that your application runs on and provides tooling for teams to build, deploy and manage their applications on that infrastructure.
-
-
-The platform is operated by [GDS Reliability Engineering](https://reliability-engineering.cloudapps.digital/) according to the [Technology & Operations Shared Responsibility Model](https://reliability-engineering.cloudapps.digital/documentation/strategy-and-principles/techops-shared-responsibility-model.html)
-
-## Who is the platform for?
-
-The platform is for teams working in the [Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) that need to run software applications.
 
 ## Features
 
 - A declarative continuous delivery workflow - merging to master triggers deployment to production
 - A container platform based on industry standard [Docker](https://docs.docker.com/) and [Kubernetes](https://kubernetes.io)
-- Build and release automation with [ConcourseCI](https://concourse-ci.org/)
+- Build and release automation powered by [ConcourseCI](https://concourse-ci.org/)
 - A private container registry with [Docker Registry](https://docs.docker.com/registry/)
 - Signing of docker image integrity with [Docker Notary](https://docs.docker.com/notary/)
 - Scanning of docker images for security vulnerabilities with [clair](https://github.com/coreos/clair)
 - Monitoring and alerting with [Prometheus](https://prometheus.io/), [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) and [Grafana](https://grafana.com/)
 - Secure git-based secrets configuration with [sealed-secrets](https://github.com/bitnami-labs/sealed-secrets)
 - Ingress management with [nginx ingress controller](https://kubernetes.github.io/ingress-nginx/)
```

Comment:
remove the nginx ingress line and replace with

- Ingress management and service mesh with [istio](https://istio.io/)